helm/app-exporter/templates/servicemonitor.yaml

```diff
@@ -9,10 +8,9 @@ metadata:
 spec:
   endpoints:
     - port: web
-      scrapeTimeout: {{ .Values.scrape.timeout }}
-      interval: {{ .Values.scrape.interval }}
+      scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
```

Comment:
Those changes make sense though

Reply:
I am trying to align naming across all apps :)